views/partials/select-server-table.hbs

```diff
@@ -16,7 +16,7 @@
         <div class="selectServer__serverURL">{{server.displayName}}</div>
         <button
           class="aui-button aui-button-label select-server"
-          data-installation-id="{{installationId}}"
+          data-identifier="{{server.displayName}}"
```

Comment:
Hey sorry I still don't get it, above apps listing page is expecting an url
```
{
			key: "github-list-server-apps-page",
			name: {
				value: "GitHub Enterprise Server Apps"
			},
			url: "/jira/ghe-servers/apps?serverUrl={ac.serverUrl}",
```

But here we are supplying a name? How does that work?

Reply:
In the frontend, for GHE server, we are displaying the `baseUrl` as the identifier, whereas for GH server apps, we are displaying the `gitHubAppName` as the identifier.
So, the `name` in this case is the `baseUrl`.